components/blog/Post.tsx

```diff
@@ -0,0 +1,88 @@
+import Image from 'next/image';
+import { getStrapiMedia, formatDate } from '../../utils/blog/api-helpers';
+import { postRenderer } from '../../utils/blog/post-rendered';
+
+export interface Article {
```

Comment:
Is this `Article` interface different from the `Article` interface defined in `PostList` component?
If yes, then please only define it once.
If not, shouldn't we use different names to avoid any confusion (especially when these interfaces are imported somewhere else).

Reply:
@tapishr  ye nahi karoge? Reason bata do.